scripts/proxy.py

```diff
@@ -4,54 +4,79 @@
 import sys
 import inspect
 import traceback
+import sj
+import typing
 
-from distutils.version import LooseVersion
+from mitmproxy import http
+from mitmproxy import ctx
+from mitmproxy.script import concurrent
 
 from subprocess import CalledProcessError, Popen, PIPE, STDOUT
 
-p = Popen(['mitmdump --version'], stdout=PIPE, stdin=PIPE, stderr=STDOUT, shell=True)
+'''p = Popen(['mitmdump --version'], stdout=PIPE, stdin=PIPE, stderr=STDOUT, shell=True)
 stdout = p.communicate()[0]
-mitmversion = stdout.decode()[9:] # remove "mitmdump "
+mitmversion = stdout.decode()[9:] # remove "mitmdump "'''
 
 
-if LooseVersion(mitmversion) >= LooseVersion("0.17"):
+'''if LooseVersion(mitmversion) >= LooseVersion("0.17"):
     from mitmproxy.script import concurrent
 else:
-    from libmproxy.script import concurrent
+    from libmproxy.script import concurrent'''
 
 filename = inspect.getframeinfo(inspect.currentframe()).filename
 JALANGI_HOME = os.path.abspath(os.path.join(os.path.dirname(os.path.abspath(filename)), os.pardir))
 WORKING_DIR = os.getcwd()
 
 sys.path.insert(0, JALANGI_HOME+'/scripts')
-import sj
 
 print('Jalangi home is ' + JALANGI_HOME)
 print('Current working directory is ' + WORKING_DIR)
 
+global jalangiArgs  
 jalangiArgs = ''
+global useCache 
 useCache = True
 ignore = []
 
+# For enabling/disabling instrumentation cache (enabled by default)
```

Comment:
Are these new options?  If so, maybe we should document in the README?

Reply:
This option was existing.